resources/guest_configs/patches/0002-ci-adjust-6.1-guest-kernel-configs-for-our-CI.patch

```diff
@@ -0,0 +1,76 @@
+From 911b916fdc53acfc7229302b029fd615f1e670cf Mon Sep 17 00:00:00 2001
+From: Babis Chalios <bchalios@amazon.es>
+Date: Fri, 23 Aug 2024 12:54:26 +0200
+Subject: [PATCH 2/2] ci: adjust 6.1 guest kernel configs for our CI
```

Comment:
I was thinking this could be combined with the other patch, but actually I prefer like you have done it here, it may be better to split this per gues kernel version (4.14, 5.10, 6.1) so we can drop them as we deprecate them. Though editing does not seem that hard.

Reply:
I actually did it like that because the changes happened in different points in time (not from the same initial commit), and I thought it'd be nicer to have them in different patches (easier to revert, etc.).